plugin/al/lib/AL_USDMaya/CMakeLists.txt

```diff
@@ -201,7 +201,7 @@ target_link_libraries(${LIBRARY_NAME}
     usdImagingGL
     vt
     Boost::python
-    $<IF:$<BOOL:${IS_WINDOWS}>,Boost::thread,${Boost_THREAD_LIBRARY}>
+    Boost::thread
```

Comment:
Because of my change in PR #602 , we should now use the proper boost target name here. This is needed for python3 builds where Boost_THREAD_LIBRARY is not defined.